D:\Mindustry-Server\boing\src\bot.js:38
    if (chatRelay && message.channel.name == client.channels.cache.find(ch => ch.name === userSettings.chatChannel).name && message.author.id != client.user.id) {
                                                                                                                   ^

TypeError: Cannot read property 'name' of undefined
    at Client.<anonymous> (D:\Mindustry-Server\boing\src\bot.js:38:116)
    at Client.emit (events.js:315:20)
    at MessageCreateAction.handle (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\actions\MessageCreate.js:31:14)
    at Object.module.exports [as MESSAGE_CREATE] (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\handlers\MESSAGE_CREATE.js:4:32)
    at WebSocketManager.handlePacket (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:384:31)
    at WebSocketShard.onPacket (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketShard.js:444:22)
    at WebSocketShard.onMessage (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketShard.js:301:10)
    at WebSocket.onMessage (D:\Mindustry-Server\boing\node_modules\ws\lib\event-target.js:132:16)
    at WebSocket.emit (events.js:315:20)
    at Receiver.receiverOnMessage (D:\Mindustry-Server\boing\node_modules\ws\lib\websocket.js:834:20)
(node:1168) UnhandledPromiseRejectionWarning: Error [TOKEN_INVALID]: An invalid token was provided.
    at Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:206:52)
    at Object.<anonymous> (D:\Mindustry-Server\boing\src\bot.js:86:8)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
    at internal/main/run_main_module.js:17:47
(Use `node --trace-warnings ...` to show where the warning was created)
(node:1168) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:1168) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:7356) UnhandledPromiseRejectionWarning: Error [TOKEN_INVALID]: An invalid token was provided.
    at WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:133:26)
    at Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:21)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:7356) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:7356) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:6876) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6876) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:6876) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:7116) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:7116) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:7116) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:6468) UnhandledPromiseRejectionWarning: Error [TOKEN_INVALID]: An invalid token was provided.
    at WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:133:26)
    at Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:21)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6468) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:6468) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:6332) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6332) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:6332) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:15936) UnhandledPromiseRejectionWarning: Error [TOKEN_INVALID]: An invalid token was provided.
    at WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:133:26)
    at Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:21)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:15936) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:15936) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:5464) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:5464) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:5464) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:5968) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:5968) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:5968) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:6768) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6768) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:6768) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
internal/child_process.js:403
    throw errnoException(err, 'spawn');
    ^

Error: spawn EROFS
    at ChildProcess.spawn (internal/child_process.js:403:11)
    at Object.spawn (child_process.js:553:9)
    at Object.<anonymous> (D:\Mindustry-Server\boing\src\mserver.js:6:29)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (D:\Mindustry-Server\boing\src\bot.js:5:17) {
  errno: -4043,
  code: 'EROFS',
  syscall: 'spawn'
}
internal/child_process.js:403
    throw errnoException(err, 'spawn');
    ^

Error: spawn EROFS
    at ChildProcess.spawn (internal/child_process.js:403:11)
    at spawn (child_process.js:553:9)
    at Object.execFile (child_process.js:237:17)
    at exec (child_process.js:158:25)
    at write (C:\Users\tyler\AppData\Roaming\npm\node_modules\node-windows\lib\eventlog.js:67:3)
    at logger.value [as warn] (C:\Users\tyler\AppData\Roaming\npm\node_modules\node-windows\lib\eventlog.js:185:9)
    at launch (C:\Users\tyler\AppData\Roaming\npm\node_modules\node-windows\lib\wrapper.js:143:18)
    at process.<anonymous> (C:\Users\tyler\AppData\Roaming\npm\node_modules\node-windows\lib\wrapper.js:201:5)
    at process.emit (events.js:315:20)
    at process._fatalException (internal/process/execution.js:163:25) {
  errno: -4043,
  code: 'EROFS',
  syscall: 'spawn'
}
(node:7344) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:7344) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:7344) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:6548) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6548) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:6548) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:6312) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6312) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:6312) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:6520) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6520) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:6520) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:6128) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6128) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:6128) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:6424) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6424) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:6424) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:6004) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6004) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:6004) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:5768) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:5768) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:5768) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:5940) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 data listeners added to [Socket]. Use emitter.setMaxListeners() to increase limit
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6404) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6404) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:6404) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:6292) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6292) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:6292) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
internal/child_process.js:403
    throw errnoException(err, 'spawn');
    ^

Error: spawn EROFS
    at ChildProcess.spawn (internal/child_process.js:403:11)
    at Object.spawn (child_process.js:553:9)
    at Object.<anonymous> (D:\Mindustry-Server\boing\src\mserver.js:6:29)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (D:\Mindustry-Server\boing\src\bot.js:5:17) {
  errno: -4043,
  code: 'EROFS',
  syscall: 'spawn'
}
internal/child_process.js:403
    throw errnoException(err, 'spawn');
    ^

Error: spawn EROFS
    at ChildProcess.spawn (internal/child_process.js:403:11)
    at spawn (child_process.js:553:9)
    at Object.execFile (child_process.js:237:17)
    at exec (child_process.js:158:25)
    at write (C:\Users\tyler\AppData\Roaming\npm\node_modules\node-windows\lib\eventlog.js:67:3)
    at logger.value [as warn] (C:\Users\tyler\AppData\Roaming\npm\node_modules\node-windows\lib\eventlog.js:185:9)
    at launch (C:\Users\tyler\AppData\Roaming\npm\node_modules\node-windows\lib\wrapper.js:143:18)
    at process.<anonymous> (C:\Users\tyler\AppData\Roaming\npm\node_modules\node-windows\lib\wrapper.js:201:5)
    at process.emit (events.js:315:20)
    at process._fatalException (internal/process/execution.js:163:25) {
  errno: -4043,
  code: 'EROFS',
  syscall: 'spawn'
}
(node:6088) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 data listeners added to [Socket]. Use emitter.setMaxListeners() to increase limit
(Use `node --trace-warnings ...` to show where the warning was created)
(node:5980) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:5980) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:5980) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:5192) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:5192) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:5192) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:5972) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:5972) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:5972) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:1564) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:1564) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:1564) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
internal/child_process.js:403
    throw errnoException(err, 'spawn');
    ^

Error: spawn EROFS
    at ChildProcess.spawn (internal/child_process.js:403:11)
    at spawn (child_process.js:553:9)
    at Object.execFile (child_process.js:237:17)
    at exec (child_process.js:158:25)
    at write (C:\Users\tyler\AppData\Roaming\npm\node_modules\node-windows\lib\eventlog.js:67:3)
    at logger.value [as warn] (C:\Users\tyler\AppData\Roaming\npm\node_modules\node-windows\lib\eventlog.js:185:9)
    at launch (C:\Users\tyler\AppData\Roaming\npm\node_modules\node-windows\lib\wrapper.js:143:18)
    at process.<anonymous> (C:\Users\tyler\AppData\Roaming\npm\node_modules\node-windows\lib\wrapper.js:201:5)
    at process.emit (events.js:315:20)
    at process._fatalException (internal/process/execution.js:163:25) {
  errno: -4043,
  code: 'EROFS',
  syscall: 'spawn'
}
(node:6008) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6008) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:6008) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:6540) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6540) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:6540) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:6492) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6492) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:6492) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:6096) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6096) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:6096) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:6932) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6932) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:6932) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:6496) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6496) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:6496) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:6260) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6260) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:6260) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:6588) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6588) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:6588) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:6088) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 data listeners added to [Socket]. Use emitter.setMaxListeners() to increase limit
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6484) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6484) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:6484) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:6476) UnhandledPromiseRejectionWarning: FetchError: request to https://discord.com/api/v7/gateway/bot failed, reason: getaddrinfo ENOTFOUND discord.com
    at RequestHandler.execute (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:93:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async RequestHandler.push (D:\Mindustry-Server\boing\node_modules\discord.js\src\rest\RequestHandler.js:39:14)
    at async WebSocketManager.connect (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\websocket\WebSocketManager.js:138:9)
    at async Client.login (D:\Mindustry-Server\boing\node_modules\discord.js\src\client\Client.js:223:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6476) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:6476) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
